packages/ui/src/NumberBadge.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react'
+
+import styles from './NumberBadge.module.scss'
+
+export type NumberBadgeProps = {
+  number: number
```

Comment:
nit: Should we also add `size` here?

Reply:
There is only one site in ZeroHeight. AFAIK, we'll only use the `NumberBadge` for side menu, so there should not be need for that at the moment.